test/controllers/graphql_controller_3_test.rb

```diff
@@ -1589,6 +1589,27 @@ def setup
     assert_match /Sorry/, @response.body
   end
 
+
+test "should return suggested similar items" do
```

Comment:
Can you please fix the indentation here? :)

Reply:
I don't see the indentation issue you're talking about?